apply/current_cluster.go

```diff
@@ -33,23 +33,22 @@ func GetCurrentCluster() (*v1.Cluster, error) {
 	return getCurrentNodes()
 }
 
-func listNodes() *corev1.NodeList {
+func listNodes() (*corev1.NodeList, error) {
 	c, err := client.NewClientSet()
 	if err != nil {
-		logger.Info("current cluster not found, will create a new cluster %v", err)
-		return nil
+		return nil, fmt.Errorf("current cluster not found, will create a new cluster %v", err)
 	}
 	nodes, err := client.ListNodes(c)
 	if err != nil {
-		logger.Info("current cluster nodes not found, will create a new cluster")
-		return nil
+		return nil, fmt.Errorf("current cluster nodes not found, will create a new cluster %v", err)
```

Comment:
will create a new cluster should not in this function. write it in get current nodes